File: src/com/Torvald/Terrarum/Actors/Glowing.java
package com.Torvald.Terrarum.Actors;

import org.newdawn.slick.GameContainer;
import org.newdawn.slick.Graphics;

/**
 * Created by minjaesong on 16-01-25.
 */
public interface Glowing {

    void drawGlow(GameContainer gc, Graphics g);

    void updateGlowSprite(GameContainer gc, int delta_t);

}

File: src/com/Torvald/Terrarum/gameactors/Glowing.kt
package com.torvald.terrarum.gameactors

import org.newdawn.slick.GameContainer
import org.newdawn.slick.Graphics

/**
 * Created by minjaesong on 16-03-14.
 */
interface Glowing {
    fun drawGlow(gc: GameContainer, g: Graphics)

    fun updateGlowSprite(gc: GameContainer, delta: Int)
}